### DISCUSSION TITLE
finetuning mask2former instance segementation

### DISCUSSION THREAD

**Discussion by nicozhou7** [open-mmlab/mmdetection#8252] [Q&A] [answered]
Hi, I want to finetune mask2former on another human only dataset, how can I use the pretrained on coco dataset WEIGHT and finetune it on my dataset? 

I mean how to load pretrained weight to my training ?

**Comment by chhluo** [comment, marked as answer]
set `load_from` to the path to the checkpoint and update `num_things` and `num_stuff` in config file.
